launch/neo_usboard_v2.yaml

```diff
@@ -0,0 +1,4 @@
+can_device: can0
```

Comment:
I would keep `can_device` empty by default, so it's using the serial port by default.